fastchat/conversation.py

```diff
@@ -131,7 +132,15 @@ def get_prompt(self) -> str:
                 if message:
                     ret += role + ": " + message + self.sep
                 else:
-                    ret += role + ": " # must be end with a space
+                    ret += role + ": "  # must be end with a space
+            return ret
+        elif self.sep_style == SeparatorStyle.H2OGPT:
```

Comment:
Can we merge this with `SeparatorStyle .NO_COLON_SINGLE`?

Reply:
Yes, thank you for the suggestion. 
I also fixed the error that it was actually using a wrong SeparatorStyle.